app/grandchallenge/challenges/emails.py

```diff
@@ -42,3 +44,133 @@ def send_external_challenge_created_email(challenge):
         subject=f"[{site.domain.lower()}] New External Challenge",
         message=message,
     )
+
+
+def send_challenge_requested_email_to_reviewers(challengerequest):
+    site = Site.objects.get_current()
+    update_url = reverse(
+        "challenges:request-list",
+    )
+    message = (
+        f"Dear reviewers,\n\n"
+        f"User {challengerequest.creator} has just requested the challenge "
+        f"{challengerequest.title}. To review the challenge, go here:"
+        f"{update_url}\n\n"
+        f"Regards,\n"
+        f"{site.name}\n\n"
+        f"This is an automated service email from {site.domain}."
+    )
+    reviewers = (
+        Group.objects.filter(name=settings.CHALLENGE_REVIEWERS_GROUP_NAME)
+        .get()
+        .user_set.all()
+    )
+
+    send_mail(
+        subject=f"[{site.domain.lower()}] New Challenge Requested",
+        message=message,
+        from_email=settings.DEFAULT_FROM_EMAIL,
+        recipient_list=[user.email for user in reviewers],
+    )
+
+
+def send_challenge_requested_email_to_requester(challengerequest):
+    site = Site.objects.get_current()
+    addition = ""
+    if (
+        challengerequest.challenge_type
+        == challengerequest.ChallengeTypeChoices.T2
+    ):
+        budget = ""
+        for key, value in challengerequest.budget.items():
+            budget += f"{key}: {value}\n"
+        addition += (
+            f"For your type 2 challenge, we have calculated the following budget estimate "
+            f"based on the information you provided:\n"
+            f"{budget}\n\n"
+        )
+
+    message = (
+        f"Dear {challengerequest.creator.username},\n\n"
+        f"Your challenge request has been sent to the reviewers. You will "
+        f"receive an email informing you of our decision within the next 4 weeks. "
```

Comment:
4 weeks is quite a long time for challenge admins, would be good if this is quicker!

Reply:
Agreed, we;ll actually be doing the review every 2 weeks, but I think it's better to give a generous time frame in this email or else we'll get swamped by emails asking for a status update. Same as we do for verifications.